Java/measureeval/src/main/resources/application.yml

```diff
@@ -62,3 +54,10 @@ link:
         path: /api/{facilityId}/QueryResult/{correlationId}
         query:
           - queryType={queryType}
+
+secret-management:
+  key-vault-uri: ''
+
+authentication:
+  anonymous: false
+  authority: ''
```

Comment:
Those keys (secretManagement:managerUri and authentication:enableAnonymousAccess) were added also in ACA configurations as well as in application.yml with same names. Now they have different names and I do not think if that will work propertly when we switch back and forth between Azure and local based on spring.cloud.azure.appconfiguration.enabled flag.

Reply:
I'd like to stick to this (hopefully consistent) naming convention... So, when I merge these changes, I'll update the app config to use the correct properties.